packages/create-yoshi-app/templates/fullstack/typescript/index-dev.ts

```diff
@@ -0,0 +1,5 @@
+import { env } from './environment';
+
+(async () => {
+  await env.start();
+})();
```

Comment:
```suggestion
env.start();
```

I think that the `async` IIFE + `await` became redundant